cmd/dns-operator/main.go

```diff
@@ -21,11 +22,12 @@ const operatorNamespace = "openshift-dns-operator"
 func main() {
 	metricsserver.DefaultBindAddress = "127.0.0.1:60000"
 
-	// This is required because controller-runtime expects its consumers to
-	// set a logger through log.SetLogger within 30 seconds of the program's
-	// initalization. We have our own logger and can configure controller-runtime's
-	// logger to do nothing.
-	ctrlruntimelog.SetLogger(logr.New(ctrlruntimelog.NullLogSink{}))
+	opts := zap.Options{
+		Development: true,
+	}
+	opts.BindFlags(flag.CommandLine)
+	flag.Parse()
+	ctrl.SetLogger(zap.New(zap.UseFlagOptions(&opts)))
```

Comment:
As per previous discussion with @Miciah and @alebedev87, setting logger to controller-runtime log to enable logging from dnsnameresolver controller. The future plan is to change existing logrus logging to controller-runtime logging.